cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/exceptions/UpdateException.java

```diff
@@ -0,0 +1,11 @@
+package cloud.foundry.cli.crosscutting.exceptions;
+
+/**
+ * Signals that a update operation has failed.
+ */
+public class UpdateException extends Exception {
```

Comment:
When selecting RuntimeException instead of Exception as super class, you are not forced to use throws/a try-catch block when calling a method that might throw an Update Exception. This can reduce boilerplate code (since we have centralized exception handling)

Reply:
Full ack. Please consider adding the exception there when converting it to an unchecked one.

Reply:
Changed.